.github/workflows/issue-regression-labeler.yml

```
@@ -0,0 +1,32 @@
+# Apply potential regression label on issues
+name: issue-regression-label
+on:
+  issues:
+    types: [opened, edited]
+jobs:
```

Let's set global perms for the workflow to baseline for OSSF compliance.

```suggestion
permissions: read-all
jobs:
```

---

Done